tests/fast-integration/README.md

```diff
@@ -63,6 +63,13 @@ npm install
 npm run test-eventing
 ```
 
+**NOTE:** You can change the namespace used for the Commerce Mock objects using:
+```
+MOCK_NAMESPACE=mock2 make ci-test-eventing
+```
+Using a different namespace changes the host used for the Commerce Mock APIRule as well. 
+This allows different Commerce Mocks to co-exist in different namespaces.
```

Comment:
```suggestion
> **NOTE:** You can change the Namespace used for the Commerce Mock objects using:
> ```
> MOCK_NAMESPACE=mock2 make ci-test-eventing
> ```
> Using a different Namespace changes the host used for the Commerce Mock APIRule as well. 
> This allows different Commerce Mocks to co-exist in different Namespaces.
```

As per our guidelines, we [capitalize `Namespace`](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#capitalized-terms) and [put the NOTE, CAUTION, and TIP notes in panels](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#panels). Please check the guidelines for reference.